lib/mrkt/concerns/custom_objects.rb

```diff
@@ -0,0 +1,54 @@
+module Mrkt
+  module CustomObjects
+    def get_list_of_custom_objects(object_names)
```

Comment:
According the [Marketo documentation](http://developers.marketo.com/documentation/custom-api/get-list-of-custom-objects/) object names in custom objects listings are optional, so we should also not require that.

See the `get_leads` method and `fields` or `batch_size`, `next_page_token` parameters.